src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -2380,11 +2380,11 @@
         </summary>
       </PropertyComment>
     </Field>
-    <Field Name="OpenParenToken" Type="SyntaxToken">
+    <Field Name="OpenParenToken" Type="SyntaxToken" Optional="true">
```

Comment:
Do we want to document this as a breaking change?

Reply:
Yes